src/rgw/rgw_kafka.cc

```diff
@@ -632,6 +634,20 @@ class Manager {
       ldout(cct, 1) << "Kafka connect: user/password are only allowed over secure connection" << dendl;
       return false;
     }
+    
+    if (brokers.has_value()) {
+      std::string kafka_brokers = brokers.get();
```

Comment:
nit: this is unnecessary allocation+copym just pass `brokers.get()` to the split function